resources/views/contact.blade.php

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Basma please add this:
@push('pageHeader')
Contact
@endpush
here at the top of this file as the header shows 'Contact' not 'About us' as Nora recommended before